docs/admin-guide/troubleshooting.md

```diff
@@ -93,114 +88,89 @@ The steps below are applicable for Sev 1 & 2 incidents, see the Escalations sect
-| --------------- | ---- |
-| Slow Performace |      |
-| Date Loss       |      |
-| Key Exposure    |      |
-| Failover        |      |
-
-
-
-### :harbor: Harbor
-
-|                 |      |
-| --------------- | ---- |
-| Slow Performace |      |
-| Date Loss       |      |
-| Key Exposure    |      |
-| Failover        |      |
-
-
-
-### :elastic: Elasticsearch
-
-|                 |      |
-| --------------- | ---- |
-| Slow Performace |      |
-| Date Loss       |      |
-| Key Exposure    |      |
-| Failover        |      |
-
-
-
-
-
+|                               |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Deployments                   | See [Troubleshooting Deployments](https://learnk8s.io/troubleshooting-deployments) <br>:bash: [kubectl-debug](https://github.com/aylei/kubectl-debug)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
My general take is that we should avoid `<a>` tags unless the result cannot be achieved in raw markdown (for example certain table formats). I have converted the links to standard markdown.